### DISCUSSION TITLE
[@types/node] missing `assert` from `TestContext`

### DISCUSSION THREAD

**Discussion by electrovir** [DefinitelyTyped/DefinitelyTyped#70134] [Issues with a @types package] [answered]
Node.js docs on `TestContext.assert` are here: https://nodejs.org/api/test.html#contextassert

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
It will be added in `v22.2.0` typings

**Reply by Semigradsky** [reply, marked as answer]
Done

**Reply by electrovir** [reply]
Thank you! Was `TestContext.assert.snapshot` not added because it's experimental still?

**Reply by Semigradsky** [reply]
`snapshot` will be added in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70208
